@shared/api/external.ts

```diff
@@ -244,28 +277,9 @@ export const setAllowedStatus = async (): Promise<boolean> => {
     console.error(e);
   }
 
-  const { isAllowed, error } = response;
-
-  if (error) {
-    throw error;
-  }
-  return isAllowed;
-};
-
-export const requestUserInfo = async (): Promise<UserInfo> => {
```

Comment:
I'm removing this method as it's duplicative of the `getAddress` method. I'm leaving the underlying service to maintain backward compatibility